test/api/v3/integration/user/GET-user.test.js

```diff
@@ -38,4 +38,27 @@ describe('GET /user', () => {
     expect(returnedUser.notifications).to.exist;
     expect(returnedUser.stats).to.not.exist;
   });
+
+  it('returns the full inbox', async () => {
+    const otherUser = await generateUser();
+    const amountOfMessages = 12;
+
+    const allMessagesPromise = [];
+    for (let i = 0; i < amountOfMessages; i += 1) {
+      allMessagesPromise.push(
+        otherUser.post('/members/send-private-message', {
+          toUserId: user.id,
+          message: `Message Num: ${i}`,
+        }),
+      );
+    }
```

Comment:
Can be done without local mutation with `lodash/range`. Also think `Promise` should be the plural part, since it's a list of message promises, not a promise for all messages.

```suggestion
    const allMessagePromises = range(amountOfMessages)
      .map(i => otherUser.post('/members/send-private-message', {
        toUserId: user.id,
        message: `Message Num: ${i}`,
      }));
```

Or if you want to get fancy about it:
```javascript
    const allMessagePromises = range(amountOfMessages)
      .map(i => `Message Num: ${i}`)
      .map(message => ({ toUserId: user.id, message }))
      .map(body => otherUser.post('/members/send-private-message', body));
```

Once you've got that statement down to a single statement you might even consider inlining it in the `await Promise.all(/*...*/);`, but I'm not sure if I like that.